scripts/get_digest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
```

Comment:
add some doc here. explain the purpose of this script, the parameters and return values

Reply:
also, remove the ".sh" termination. an executable script usually does not use .sh termination

Reply:
done